tox.ini

```diff
@@ -21,7 +21,7 @@ commands = flake8 --max-complexity=8 pipeline --exclude='test/output' test
 [testenv:pylint-errors]
 deps = pylint
        -r{toxinidir}/requirements.txt
-commands = pylint -f colorized -E pipeline --ignore test/output test
+commands = pylint -f colorized -E pipeline --ignore test
```

Comment:
I don't think we want this -- the tests themselves should still be linted. I will try to find an alternative solution to the build failure in #45.